src/test/scala/com/fulcrumgenomics/umi/GroupReadsByUmiTest.scala

```diff
@@ -267,6 +268,8 @@ class GroupReadsByUmiTest extends UnitSpec with OptionValues with PrivateMethodT
 
       hist.toFile.exists() shouldBe true
 
+      metrics.toFile.exists() shouldBe true
```

Comment:
You should assert the contents of this file are equal to an expected Metric.

You can do this with `val expected: UmiGroupingMetric = ...` and a call to `Metric.read()`.